config/initializers/highlight_override.rb

```diff
@@ -60,39 +60,39 @@ def create_model!(user_uuid:)
 end
 
 Api::V0::Bindings::Highlights.class_exec do
-  def self.create_from_models(highlights, pagination, total_count)
+  def self.create_from_models(highlights)
     highlights_bindings = highlights.map do |highlight|
       Api::V0::Bindings::Highlight.create_from_model(highlight)
     end
-    new(meta: pagination.merge(total_count: total_count),
+    new(meta: {
+          total_count: highlights.count,
+        },
         data: highlights_bindings)
   end
 end
 
 Api::V0::Bindings::GetHighlights.class_exec do
-  PAGING_DEFAULTS = {
-    per_page: 15,
-    page: 1
-  }.freeze
-
-  FILTER_DEFAULTS = {
-    order: 'asc'
-  }.freeze
-
   def query(user_uuid:)
     highlights = ::Highlight.by_user(user_uuid)
 
     # The submitted GetHighlight properties create automatic chaining via
     # the by_X scopes on the Highlight model.
-    pagination = to_hash.dup
-    filter_by = pagination.slice!(:page, :per_page)
-    filter_by = FILTER_DEFAULTS.merge(filter_by)
-    filter_by.each do |key, value|
+    to_hash.each do |key, value|
       highlights = highlights.public_send("by_#{key}", value) if value.present?
     end
-    pagination = PAGING_DEFAULTS.merge(pagination)
 
-    [highlights.paginate(pagination), pagination, highlights.count]
+    highlights = highlights.to_a
+
+    if source_ids.present?
+      # Sort the highlights in Ruby, not Postgres
```

Comment:
seems like this comment is unnecessary...it's obvious from the code

Reply:
Haha I added this specifically because I thought you'd ask for it :-D

Reply:
nah...comments in code are almost always a code smell.  except for e.g., comments to external links....